lib/Ebook.php

```diff
@@ -529,29 +521,26 @@ protected function GetTextSinglePageUrl(): string{
 		return $this->_TextSinglePageUrl;
 	}
 
-	protected function GetTextSinglePageSizeNumber(): string{
```

Comment:
This diff looks messy in unified mode, but easier to read in split mode (gear menu > split) because I replaced two functions with one. I think this is a nice improvement and a logical step after we decided to store the byte count in the DB.